syntax = "proto3";

package test;

service MRTRpcSrv {
  rpc execute(MRTClientReq) returns(stream MRTServerResp) {}
  rpc submit(stream MRTClientReqStream) returns (stream MRTServerResp) {}
}

message MRTClientReq {
  string content = 1;
}

message MRTClientReqStream {
  bytes chunck = 1;
}

message MRTServerResp {
  string logging_str = 1;
}
